### PR TITLE
feat(cli): add --chrome-path option (#700)

### PR DESCRIPTION
**Pull request by sinedied**

### CODE REVIEW

**Comment by Zidious** [comment]
Hey @sinedied,

Thanks for the PR! Apologies the delay, we'll get get to reviewing it this week.

File: packages/cli/src/lib/webdriver.ts [hunk 1]
```diff
@@ -21,6 +21,10 @@ const startDriver = async (
       }, options);
     }
 
+    if (config.chromePath) {
+      options.setChromeBinaryPath(config.chromePath);
```
Comment (michael-siek) [inline packages/cli/src/lib/webdriver.ts:25]: I don't think we should force a user to type an absolute path here we should do something like 
```js
options.setChromeBinaryPath(path.resolve(config.chromePath));
```

**Review by michael-siek — CHANGES_REQUESTED** [review]

File: packages/cli/src/lib/webdriver.ts (hunk 1 above)
Comment (sinedied) [reply packages/cli/src/lib/webdriver.ts]: I made the change and rebased the PR, sorry for the delay

File: packages/cli/src/bin/cli.ts [hunk 2]
```diff
@@ -77,6 +77,10 @@ program
     '--chromedriver-path <path>',
     'Absolute path to the desired chromedriver executable'
   )
+  .option(
+    '--chrome-path <path>',
+    'Absolute path to the desired chrome executable'
```
Comment (straker) [inline packages/cli/src/bin/cli.ts:82]: With the last change we'll need to update the message as well.

```suggestion
    'Path to the desired chrome executable'
```

**Review by straker — CHANGES_REQUESTED** [review]

**Comment by Zidious** [comment]
Hey @sinedied, 

Just checking up on the status of this PR or you need us to take this over? 

Thanks again!

**Comment by sinedied** [comment]
Thanks @Zidious for the reminder!
I forgot about it as I was in vacation during the last notification, oups... I made the requested changes and rebased the branch, should be good now

**Comment by Zidious** [comment]
Hey @sinedied, 

We should have CI up and running on contributor PR's soon and we'll be able to merge in your PR. 

Thanks again!

**Review by straker — APPROVED** [review]

**Comment by straker** [comment]
Thanks again for the contribution.

Approved for security.

**Pull request merged**